instrumentation/jboss-logmanager/jboss-logmanager-mdc-1.1/javaagent/build.gradle.kts

```diff
@@ -13,4 +13,5 @@ muzzle {
 
 dependencies {
   library("org.jboss.logmanager:jboss-logmanager:1.1.0.GA")
+  latestDepTestLibrary("org.jboss.logmanager:jboss-logmanager:+")
```

Comment:
This actually is added automatically, there's no need to specify it manually (unless you want to limit the latest version; which is not the case here)
```suggestion
```